core-bundle/src/Resources/public/core.js

```diff
@@ -651,6 +651,7 @@ var AjaxRequest =
 	toggleFieldset: function(el, id, table) {
 		el.blur();
 		var fs = $('pal_' + id);
+		Backend.getScrollOffset();
```

Comment:
During my tests I've noticed that opening and closing legends does not store the offset. But it's actually quite nice to get back to the place you've last opened or closed a legend so I've added that here :)